src/components/screens/ErrorFallback.tsx

```diff
@@ -20,6 +20,7 @@ export function ErrorFallback() {
 
   useEffect(() => {
     analytics.page("WalletConnect ErrorBoundary Page");
+    // eslint-disable-next-line react-hooks/exhaustive-deps
   }, []);
```

Comment:
We don't want to call again the function after first load, it's effectively an onMount
Also it's mainly a revert because I had to remove this line recently when removing the eslint plugin for react hooks as it was not compatible with the latest version of eslint